types/jquery/index.d.ts

```diff
@@ -680,6 +680,14 @@ interface JQueryCoordinates {
 }
 
 /**
+ * The interface used to specify the properties parameter in css()
+ */
+type cssPropertySetter = (index: number, value: string) => string | number;
```

Comment:
From the documentation in the example, parameter `value` is optional.... I would make that parameter as such and keep the exist test case and add new one with both specified